mods/cnc/chrome/install.yaml

```diff
@@ -82,7 +82,7 @@ Container@INSTALL_PANEL:
 			Text: Use CD
 
 Container@INSTALL_FROMCD_PANEL:
-	Logic: InstallFromCDLogic
+	Logic: InstallFromTTFDLogic
```

Comment:
You probably should add a check in `InstallFromCDLogic` and let it decide whether to use `InstallFromTTFDLogic`, depending on what it finds on the CD.